apps/src/blockly/addons/cdoXml.ts

```diff
@@ -182,6 +182,7 @@ export function makeWhenRunUndeletable(blockElement: Element) {
     return;
   }
   blockElement.setAttribute('deletable', 'false');
+  blockElement.setAttribute('movable', 'false');
```

Comment:
nit: rename this function/update the comment to note it also makes the when run block immovable?